components/support/webextensions/src/main/java/mozilla/components/support/webextensions/WebExtensionSupport.kt

```diff
@@ -213,10 +214,10 @@ object WebExtensionSupport {
     suspend fun awaitInitialization() = initializationResult.await()
 
     /**
-     * Queries the [engine] for installed web extensions and adds them to the [store].
+     * Queries the [WebExtensionManager] for installed web extensions and adds them to the [store].
      */
-    private fun registerInstalledExtensions(store: BrowserStore, engine: Engine) {
-        engine.listInstalledWebExtensions(
+    private fun registerInstalledExtensions(store: BrowserStore, manager: WebExtensionManager) {
```

Comment:
We'd need to also switch to the new type in `initialize` here if we want to include this class in the refactoring and update the KDocs.

Reply:
Fixed.